src/domain/orders/details/returns/index.js

```diff
@@ -136,219 +149,328 @@ const ReturnMenu = ({ order, onReturn, onDismiss, toaster }) => {
+                    <>
+                      <span
+                        className="mr-2 cursor-pointer text-grey-40"
+                        onClick={() => setRefundEdited(true)}
+                      >
+                        <EditIcon size={20} />{" "}
+                      </span>
+                      {`${displayAmount(
+                        order.currency_code,
+                        refundAmount
+                      )} ${order.currency_code.toUpperCase()}`}
+                    </>
+                  )}
+                </div>
+              </div>
+              {refundEdited && (
                 <CurrencyInput
-                  currency={order.currency_code}
-                  value={refundAmount / 100}
-                  onChange={handleRefundUpdated}
-                />
-              </Box>
-            </Flex>
+                  className="mt-2"
+                  size="small"
+                  currentCurrency={order.currency_code}
+                  readonly
+                >
+                  <CurrencyInput.AmountInput
+                    label={"Amount"}
+                    amount={refundAmount}
+                    onChange={handleRefundUpdated}
+                  />
+                </CurrencyInput>
+              )}
+            </div>
           )}
         </Modal.Content>
-        <Modal.Footer justifyContent="space-between">
-          <Flex>
-            <Box px={0} py={1}>
+        <Modal.Footer>
+          <div className="flex w-full justify-between">
+            <div
+              className="items-center h-full flex cursor-pointer"
+              onClick={() => setNoNotification(!noNotification)}
+            >
+              <div
+                className={`w-5 h-5 flex justify-center text-grey-0 border-grey-30 border rounded-base ${
+                  !noNotification && "bg-violet-60"
+                }`}
+              >
+                <span className="self-center">
+                  {!noNotification && <CheckIcon size={16} />}
+                </span>
+              </div>
               <input
                 id="noNotification"
+                className="hidden"
                 name="noNotification"
                 checked={!noNotification}
                 onChange={() => setNoNotification(!noNotification)}
                 type="checkbox"
               />
-            </Box>
-            <Box px={2} py={1}>
-              <Text fontSize={1}>Send notifications</Text>
-            </Box>
-          </Flex>
-          <Button loading={submitting} type="submit" variant="primary">
-            Complete
-          </Button>
+              <span className="ml-3 flex items-center text-grey-90 gap-x-xsmall">
+                Send notifications
+                <InfoTooltip content="Notify customer of created return" />
+              </span>
+            </div>
+            <div className="flex gap-x-xsmall">
+              <Button
+                loading={submitting}
```

Comment:
i'd say rm but if you think we should block the user from dismissing the modal, maybe disabling the button would make more sense as opposed to showing a spinner .. what do you think?